arch/riscv/Kconfig

```diff
@@ -225,6 +225,15 @@ config RISCV_RESERVED_IRQ_ISR_TABLES_OFFSET
 	  properly sized to take into account this offset. This is a hidden
 	  option which needs to be set per architecture and left alone.
 
+config RISCV_EMPTY_IRQ_SPURIOUS
+	bool "Create empty spurious interrupt handler"
+	default y if NRFE
+	default n
```

Comment:
`bool` symbols implicitly default to n. Therefore, `default n` are (almost) always redundant.